### PR TITLE
[MP v2] Fix all non-test related type errors

### PR DESCRIPTION
**Pull request by alloy**
📈

### CODE REVIEW

File: src/Apps/Artwork/Components/__stories__/PricingContext.story.tsx [hunk 1]
```diff
@@ -27,11 +27,12 @@ const MockPricingContext = ({
   )
 }
 
-const mockArtwork: PricingContext_artwork = {
-  " $refType": null,
-  artists: [{ slug: "alex-katz" }],
+const mockArtwork: PricingContextStoryQueryRawResponse["artwork"] = {
+  id: "opaque-artwork-id",
```
Comment (mzikherman) [inline src/Apps/Artwork/Components/__stories__/PricingContext.story.tsx]: nice

File: src/Artsy/Relay/RootQueryRenderer.tsx [hunk 2]
```diff
@@ -36,12 +26,15 @@ const RendererWithContext = withSystemContext(Renderer)
...
-export const RootQueryRenderer: React.SFC<Props> = ({
-  user,
-  children,
-  ...props
-}) => (
-  <SystemContextProvider user={user}>
-    <RendererWithContext {...props} />
-  </SystemContextProvider>
-)
+export class RootQueryRenderer<T extends OperationType> extends React.Component<
```
Comment (mzikherman) [inline src/Artsy/Relay/RootQueryRenderer.tsx]: not enough `extends` nesting, needs at least 6 more :)

File: src/Components/ArtworkFilter/Dropdown.tsx [hunk 3]
```diff
@@ -47,10 +47,10 @@ export class Dropdown extends React.Component<DropdownProps, DropdownState> {
     this.props.onSelected(slice, value)
   }
 
-  getSelectedName(id) {
+  getSelectedName(value) {
     const selectedCount = find(
       this.props.aggregation.counts,
-      count => count.id === id
+      count => count.value === value
```
Comment (mzikherman) [inline src/Components/ArtworkFilter/Dropdown.tsx]: Ah, good one.

File: src/Components/Gene/index.tsx [hunk 4]
```diff
@@ -182,10 +181,11 @@ class GeneContents extends React.Component<Props, State> {
         variables={{
           geneID,
           medium,
-          price_range,
-          sort,
-          for_sale,
-          dimension_range,
+          priceRange: price_range,
+          // FIXME: Sorting is broken
```
Comment (mzikherman) [inline src/Components/Gene/index.tsx]: Yea, confirmed in prod. Weird.

**Comment by artsyit** [comment]
:rocket: PR was released in v22.0.0 :rocket:

**Pull request merged**